### PR TITLE
Add Docs

### PR DESCRIPTION
**Pull request by naveen521kk**
A sphinx based docs setup. Should be useful in future, but let it be for now.

### CODE REVIEW

**Comment by naveen521kk** [comment]
View docs at https://manimpango--10.org.readthedocs.build/en/10/

File: manimpango/cmanimpango.pyx [hunk 1]
```diff
@@ -144,13 +144,17 @@ class PangoUtils:
 
     @staticmethod
     def remove_last_M(file_name: str) -> None:
+        """Formatting done for the ``SVG`` file."""
```
Comment (PhilippImhof) [inline manimpango/cmanimpango.pyx:147]:
```suggestion
        """Remove element from the SVG file in order to allow comparison."""
```

File: manimpango/cmanimpango.pyx [hunk 2]
```diff
@@ -144,13 +144,17 @@ class PangoUtils:
...
         with open(file_name, "r") as fpr:
             content = fpr.read()
         content = re.sub(r'Z M [^A-Za-z]*? "\/>', 'Z "/>', content)
         with open(file_name, "w") as fpw:
             fpw.write(content)
 
 class TextSetting(object):
+    """Formatting done for array of character in
+    :py:class:`manim.mobject.svg.text_mobject.Text`
+    """
```
Comment (PhilippImhof) [inline manimpango/cmanimpango.pyx:157]:
```suggestion
    """Formatting for slices of a :py:class:`manim.mobject.svg.text_mobject.Text` object."""
```

File: manimpango/cmanimpango.pyx [hunk 3]
```diff
@@ -170,6 +174,9 @@ def text2svg(
     height:int,
     orig_text:str
 ) -> int:
+    """Main function called from Manim to render a
+    :py:class:`manim.mobject.svg.text_mobject.Text` to a SVG file.
+    """
```
Comment (PhilippImhof) [inline manimpango/cmanimpango.pyx:179]:
```suggestion
    """Main function called from Manim to render an SVG file from a 
    :py:class:`manim.mobject.svg.text_mobject.Text` object.
    """
```

File: manimpango/cmanimpango.pyx [hunk 4]
```diff
@@ -279,6 +286,9 @@ class MarkupUtils:
         width: int,
         height: int,
     ) -> int:
+        """Main fuction called from Manim, to convert
+        :py:class:`manim.mobject.svg.text_mobject.MarkupText` to SVG file.
+        """
```
Comment (PhilippImhof) [inline manimpango/cmanimpango.pyx:291]:
```suggestion
        """Main function called from Manim, to render an SVG file from a 
        :py:class:`manim.mobject.svg.text_mobject.MarkupText` object.
        """
```

**Review by PhilippImhof — CHANGES_REQUESTED** [review]
some suggestions

**Comment by naveen521kk** [comment]
If it's ok @PhilippImhof can you give an approve?

File: manimpango/cmanimpango.pyx [hunk 5]
```diff
@@ -144,13 +144,15 @@ class PangoUtils:
...
     def remove_last_M(file_name: str) -> None:
+        """Remove element from the SVG file in order to allow comparison."""
         with open(file_name, "r") as fpr:
             content = fpr.read()
         content = re.sub(r'Z M [^A-Za-z]*? "\/>', 'Z "/>', content)
         with open(file_name, "w") as fpw:
             fpw.write(content)
 
 class TextSetting(object):
+    """Formatting for slices of a :py:class:`manim.mobject.svg.text_mobject.Text` object."""
```
Comment (leotrs) [inline manimpango/cmanimpango.pyx:155]: Nitpick: `:py:` is redundant:
```suggestion
    """Formatting for slices of a :class:`manim.mobject.svg.text_mobject.Text` object."""
```

File: manimpango/cmanimpango.pyx [hunk 6]
```diff
@@ -170,6 +172,9 @@ def text2svg(
     height:int,
     orig_text:str
 ) -> int:
+    """Main function called from Manim to render an SVG file from a
+    :py:class:`manim.mobject.svg.text_mobject.Text` object.
+    """
```
Comment (leotrs) [inline manimpango/cmanimpango.pyx:177]:
```suggestion
    """Render an SVG file from a :class:`manim.mobject.svg.text_mobject.Text` object."""
```

File: manimpango/cmanimpango.pyx [hunk 7]
```diff
@@ -284,6 +289,9 @@ class MarkupUtils:
         width: int,
         height: int,
     ) -> int:
+        """Main function called from Manim, to render an SVG file from a
+        :py:class:`manim.mobject.svg.text_mobject.MarkupText` object.
+        """
```
Comment (leotrs) [inline manimpango/cmanimpango.pyx:294]:
```suggestion
        """Render an SVG file from a :class:`manim.mobject.svg.text_mobject.MarkupText` object."""
```

**Review by leotrs — CHANGES_REQUESTED** [review]

**Review by leotrs — APPROVED** [review]

**Comment by naveen521kk** [comment]
I think @leotrs you don't have write access to this repo. or else it should have been merged by now.

**Comment by leotrs** [comment]
I believe I do have access, I think I just forgot to merge after accepting :sweat_smile:

**Comment by naveen521kk** [comment]
I had enabled auto-merge which means, it should I have merged if you had writes access.

**Comment by leotrs** [comment]
Oh snap, then I don't :thinking:

**Pull request merged**